### PR TITLE
body-border

### PR DESCRIPTION
**Pull request by msarroca**

### CODE REVIEW

**Comment by Ury18** [comment]
Mis 10

File: index.html [hunk 1]
```diff
@@ -7,17 +7,10 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
 </head>
 
-<body>
+
```
Comment (Ury18) [inline index.html:10]: Si

File: index.html [hunk 2]
```diff
@@ -7,17 +7,10 @@
...
   <title>Hello World</title>
   </head>
-
-
-  <body style="background:purple">
-    
-
+  <body style="background:purple; border:2px solid orange ">
```
Comment (Ury18) [inline index.html]: Red too man

Comment (msarroca) [reply index.html]: Yeah men

**Pull request merged**